viv_utils/emulator_drivers.py

```diff
@@ -31,67 +30,39 @@ def __str__(self):
         return "InstructionRangeExceededError(ended at instruction 0x%08X)" % self.pc
 
 
-class Hook(LoggingObject):
-    def __init__(self):
-        super(Hook, self).__init__()
-
+class Hook:
     def hook(self, callname, emu, callconv, api, argv):
         # must return something other than None if handled
         # raise UnsupportedFunction to pass
         raise UnsupportedFunction()
 
 
-class Monitor(vivisect.impemu.monitor.EmulationMonitor, LoggingObject):
-    def __init__(self, vw):
```

Comment:
breaking change: monitor no longer takes the `vw` (access via `emu.vw`)

Reply:
gotta take note of that!